src/app/pages/project/project.module.ts

```diff
@@ -99,7 +99,7 @@ import { ImportService } from 'src/app/services/import/import.service';
     IssueListComponent,
     IssueViewComponent,
     StepsListComponent,
-    ImportComponent
+    ImportComponent,
```

Comment:
have not you missed here something? I mean you leave comma at the end

Reply:
removed ,